scripts/distribute_bpe_reward.py

```diff
@@ -2,6 +2,22 @@
 
 """
 To distribute token-level rewards to sub-word units.
+
+Example: 
```

Comment:
That makes it clearer.

To be exact, it's not the file from database, but a line from a text file split at whitespaces and analogously for the reward file.

Reply:
See changes